src/Sentry/Internal/Extensions/EnumerableExtensions.cs

```diff
@@ -0,0 +1,8 @@
+namespace Sentry.Internal.Extensions;
+
+internal static class EnumerableExtensions
+{
+    public static IEnumerable<KeyValuePair<string, string>> WithValues(this IEnumerable<KeyValuePair<string, string?>> items) =>
```

Comment:
It was actually so that I could pass this into `BaggageHeader.Create` in the first place. The constructor is expecting `IEnumerable<KeyValuePair<string, string>>` whereas what we get from `data.Baggage` is a `IEnumerable<KeyValuePair<string, string?>>`.

I assume we can't change the signature on `BaggageHeader.Create` as it's public... that would be a breaking change (even if we're making it less restrictive)?

Reply:
But `BaggageHeader` is an internal class. I think it's fine to change the signature of `Create` to accept `IEnumerable<KeyValuePair<string, string?>>` and let it figure out what to do with it.

Reply:
Ahh... now it's me who didn't read things properly! 😉 

Cool, in that case I'll move it to the constructor.